pom.xml

```diff
@@ -78,6 +78,12 @@
       <groupId>com.nimbusds</groupId>
       <artifactId>nimbus-jose-jwt</artifactId>
       <version>8.0</version>
+      <exclusions>
```

Comment:
I figured out we can just exclude json-smart from nimbus because none of the nimbus functions we use end up using json-smart.